app/src/components/icon/custom/lang/fr.svg

```diff
@@ -0,0 +1,5 @@
+<svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 640 480">
```

Comment:
I think I'll have to redesign the language select and [remove the flags](https://usersnap.com/blog/design-language-switch/). But for now it looks good!

Reply:
Or use a library ... I could do that if you want.

Reply:
I think the new Dropdown component might be sufficient. I guess there are more important tasks to move this project forward, though 😉